src/index.js

```diff
@@ -535,80 +471,69 @@ class KadDHT extends EventEmitter {
 
     this._log('findPeer %s', id.toB58String())
 
-    this.findPeerLocal(id, (err, pi) => {
-      if (err) {
-        return callback(err)
-      }
+    const pi = await this.findPeerLocal(id)
```

Comment:
can we call this `pid` our `peerId` instead? I know that was before, but `pi` seems odd